### PR TITLE
SPCTR-3010: Fix - Heading highlighted text font family loading issue

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Highlighted text is not adapting the Font-family
https://brainstormforce.atlassian.net/browse/SPCTR-3010

### Screenshots

https://d.pr/v/hJ0Xu0

### How has this been tested?

- Drop heading block.
- Select any random text in that heading block.
- Select the highlight icon from the toolbar.
- Go to the stylings panel, and go to highlight styling, and add font-family.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Heading - Resolved an issue in which the font-family would not render as required for highlighted heading.
```
Comment (bsf-zanev) [inline README.md:170]:
```suggestion
* Fix: Heading - Resolved an issue in which the font-family would not render as required for highlighted headings.
```

File: src/blocks/advanced-heading/attributes.js [hunk 2]
```diff
@@ -757,6 +757,7 @@ const attributes = {
 	},
 	highLightFontWeight: {
 		type: 'string',
+		default: 'Default',
```
Comment (bsf-zanev) [inline src/blocks/advanced-heading/attributes.js:760]: The default in PHP does not match in this branch...

**Review by bsf-zanev — COMMENTED** [review]
Just one minor comment.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

File: src/blocks/advanced-heading/attributes.js (hunk 2 above)
Comment (rajmohanreddy774) [reply src/blocks/advanced-heading/attributes.js:760]: Made changes in attributes.php as per attributes.js.

**Review by bsf-zanev — APPROVED** [review]

**Review by aapatre — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**